implementations/toLowerCase.js

```diff
@@ -0,0 +1,22 @@
+function toLowerCase(str) {
+    let temp = "";
+for(let chars of str){
+  //Get the ascii value of character
+  let value = chars.charCodeAt();
+
+  //If the character is in uppercase
+  if(value >= 65 && value <= 90){
+    //convert it to lowercase
+    temp += String.fromCharCode(value + 32);
+  }else{
+    //else add the original character
+    temp += chars;
+  }
+}
+    
+}
```

Comment:
Breaking for the following case:

```javascript
const a = '-21Hello 0 World @123';
```

Reply:
Hi sorry I'm new here I refactored the code can you tell me the new PR is submitted if yes then have a look into that

Reply:
You can have a look at #7 As of now I have to close this PR as it is still incorrect and untidy.